app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None | str:
+    command_list = command.split()
```

Comment:
Unpack the split command and assign values to variables to use later so that you avoid indexing and make your code more descriptive, e.g. 
```suggestion
    command_name, input_filename, output_filename, *_ = command.split()
```